traffic_ops/app/lib/API/Server.pm

```diff
@@ -726,41 +746,26 @@ sub get_servers_by_type {
 	my $orderby           = $self->param('orderby') || "hostName";
 	my $orderby_snakecase = lcfirst( decamelize($orderby) );
 
+	my $forbidden;
```

Comment:
not sure why this changed. seems unrelated to delivery services but if you are trying to establish consistency, i am fine with it.